integreat_cms/cms/templates/organizations/organization_form.html

```diff
@@ -43,20 +60,175 @@ <h3 class="heading font-bold text-black"><i data-feather="settings" class="mr-2"
-				data-confirmation-text="{{ delete_dialog_text }}"
-				data-confirmation-subject="{{ form.name.value }}"
-				data-action="{% url 'delete_organization' region_slug=request.region.slug slug=form.instance.slug %}">
-			<i data-feather="trash-2" class="mr-2"></i>
-			{% trans 'Delete this organization' %}
-		</button>
+	{% if form.instance.id %}
+
+	{% get_current_language as LANGUAGE_CODE %}
+	{% get_language LANGUAGE_CODE as backend_language %}
+	{% with request.region.default_language as  region_default_language %}
+		<div class="grid xl:grid-cols-2 gap-4">
+			<div class="mb-4 rounded border border-solid border-blue-500 shadow-2xl bg-white">
+				<div class="p-4 rounded bg-water-500">
+				<h3 class="heading font-bold text-black"> <i data-feather="layout" class="mr-2"></i>
+					{% trans "Pages" %}
+				</h3>
+				</div>
+				<div class="table-listing w-full p-2">
+					{% if region_default_language == backend_language %}
+						
+						<table class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white">
+							
+							<thead>
+								<tr class="border-b border-solid border-gray-200">
+									<th class="text-sm text-left uppercase py-3 pl-4 pr-2">
+										{% trans "Name in " %} {{ region_default_language.translated_name }}
+									</th>
+								</tr>
+
+							</thead>
+
+							<tbody>
+								{% for page in form.instance.pages.all %}
+									{% get_translation page region_default_language.slug as page_translation %}
+									<tr class="border-b border-solid border-gray-200">
+										{% if page_translation %}
+											<td class="text-sm text-left py-3 pl-4 pr-2">
+												<a href="{% url 'edit_page' page_id=page.id region_slug=request.region.slug language_slug=region_default_language.slug %}" class="block py-1.5 px-2 overflow-hidden max-w-xs whitespace-nowrap text-ellipsis text-gray-800" title="{% trans 'View page' %}">
+													{{ page_translation.title }}
+												</a>
+											</td>
+										{% else %}
+											<td class="text-sm text-left py-3 pl-4 pr-2">
+												<i>{% trans 'Translation not available' %}</i>
+											</td>
+										{% endif %}
+									</tr>
+								{% empty %}
+								<tr>
+									<td colspan="6" class="px-4 py-3">
+										{% trans "This organization currently has no maintained pages." %}
+									</td>
+								</tr>
+							
+								{% endfor %}
+							</tbody>
+							
+						</table>
+					{% else %}
+						<table class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white">
+						
+							<thead>
+								<tr class="border-b border-solid border-gray-200">
+									<th class="text-sm text-left uppercase py-3 pl-4 pr-2">
+										{% trans "Name in " %} {{ region_default_language.translated_name }}
+									</th>
+									<th class="text-sm text-left uppercase py-3 pl-4 pr-2">
+										{% trans "Name in "%} {{ backend_language.translated_name }}
+									</th>
```

Comment:
```suggestion
									{% if region_default_language != backend_language %}
										<th class="text-sm text-left uppercase py-3 pl-4 pr-2">
											{% trans "Name in "%} {{ backend_language.translated_name }}
										</th>
									{% endif %}
```
Could you reduce code duplications by showing the default language in any case and just showing the additional column if the backend language differs? If you use the same condition in every row, I guess it's save to say that each row will have the same number of columns in the end.

Reply:
Fixed, at least I hope I did is as you intended it. The code suggestions here are a bit hard to read :D

Reply:
If it's too hard to read you can also commit the suggested changes directly via the GitHub UI and hope I didn't mess up anything :see_no_evil: